packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -567,6 +568,20 @@ export class AccountResolver {
     return { exists: false };
   }
 
+  @Mutation((returns) => BasicPayload, {
+    description: 'Consume an unblock code',
+  })
+  @CatchGatewayError
+  public async rejectUnblockCode(
+    @Args('input', { type: () => RejectUnblockCodeInput })
+    input: RejectUnblockCodeInput
+  ): Promise<BasicPayload> {
+    await Account.consumeUnblockCode(input.uid, input.unblockCode);
```

Comment:
I wonder why `unblockCodes` get normalized to all capitals. Could proably do the same here for parity.